### DISCUSSION TITLE
Gatsby

### DISCUSSION THREAD

**Discussion by tujoworker** [oven-sh/bun#2808] [General]
I see Next.js is supported. Would it be possible to get in support for Gatsby as well? 🙏 Or is there any way to run such Node.js project without the extra framework support?

**Comment by tomByrer** [comment]
Have you tried it out?

If you did & found somewhere where Bun broke node compatibility, I guess you could either figure out a workaround & share it, or see if that compatibility bug report exists already & start a new one if not.

But Gatsby is quite a beast with many layers... might want to check if Bun is comparable with whatever GraphQL libs they are using first...

**Reply by tujoworker** [reply]
Thank you Tom for you feedback 🙏

Regarding the GraphQL libs – from what I know, do they use their own Babel plugin. But Babel runs on Node.js – so I assume it would just work with bun?

I have started to look at it. But have not figured out how – should I simply call the Gatsby binary?

**Reply by ZeldOcarina** [reply]
I tried some bun install on a workspace but it doesn't seem to work, I know it's the worst feedback ever haha
